turtles-dev-community-playbook.yml

```diff
@@ -0,0 +1,32 @@
+site:
+  title: Rancher Turtles
+  url: /
+  start_page: next@turtles:en:index.adoc
+
+content:
+  sources:
+    - url: ./
+      start_paths: [versions/next]
+      branches: HEAD
+      
+# Description: SUSE UI bundle
+ui:
+  bundle:
+    url: https://github.com/rancher/product-docs-ui/blob/main/build/ui-bundle.zip?raw=true
+    snapshot: true
+  supplemental_files: ./turtles-supplemental-files
+
+asciidoc:
+  attributes:
+    build-type: community
+    page-pagination: ''
+    page-toclevels: 4@
+    tabs-sync-option: ''
+  extensions:
+    - '@asciidoctor/tabs'
+antora:
+  extensions:
+  - require: '@antora/lunr-extension'
```

Comment:
You also need `- require: '@springio/antora-xref-extension'` or you miss the extended xref validations the other playbooks have.